README.md

```diff
@@ -29,6 +29,37 @@ Run the client with:
 ```bash
 build/codex
 ```
+## Configuration
+
+It is possible to configure Codex node in several ways:
+ 1. CLI options
+ 2. Env. variable
+ 3. Config
+
+The order of priority is in the order listed above, where CLI options have highest priority when a configuration
```

Comment:
This phrasing is a little convoluted, perhaps something like:

`The order of priority is the same as above; Cli arguments > Env variables > Config file values.`